src/Ren/Data/Expression/Literal.elm

```diff
@@ -330,6 +347,43 @@ stringLiteralParser =
             ]
 
 
+{-| -}
+templateLiteralParser : Parser expression -> Parser (Literal expression)
+templateLiteralParser expressionParser =
+    let
+        isRawText c =
+            c /= '`' && c /= '\\' && c /= '$'
+    in
+    Parser.succeed Template
+        |. Parser.symbol "`"
+        |= Parser.loop []
+            (\chunks ->
+                Parser.oneOf
+                    [ Parser.succeed (\s -> Text s :: chunks)
```

Comment:
Ideally this would be merged with the previous element if it was `Text`, to avoid multiple consecutive `Text` elements in the List.